pkgs/by-name/po/pop-wallpapers/package.nix

```
@@ -3,25 +3,32 @@
   stdenvNoCC,
   fetchFromGitHub,
   imagemagick,
-  nix-update-script,
+  unstableGitUpdater,
 }:
 
 stdenvNoCC.mkDerivation (finalAttrs: {
   pname = "pop-wallpapers";
-  version = "1.0.5";
+  version = "1.0.5-unstable-2025-06-24";
 
   src = fetchFromGitHub {
     owner = "pop-os";
     repo = "wallpapers";
-    rev = "refs/tags/${finalAttrs.version}";
-    hash = "sha256-JST5Rt4Ec1lRu62PUt98S2G1vKthAyOSpyCpuCnkGmw=";
+    rev = "20a9fdd1ed86aadfbbfcd55dc3d2d9eb8ae28e15";
+    hash = "sha256-GUXxy+zRdBCsvWW9ytJZUHSw6rGc3uNTjNIsiK3l+zA=";
   };
 
+  # Future-proofs calls to imagemagick's convert feature
+  postPatch = ''
+    substituteInPlace Makefile --replace-fail "convert" "magick"
+  '';
+
   nativeBuildInputs = [ imagemagick ];
 
   makeFlags = [ "prefix=$(out)" ];
 
-  passthru.updateScript = nix-update-script { };
+  enableParallelBuilding = true;
+
+  passthru.updateScript = unstableGitUpdater { };
```

```suggestion
  passthru.updateScript = nix-update-script { extraArgs = [ "--version=branch" ]; };
```

I pretty strongly prefer `nix-update-script` for everything.

---

Applied.